src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  }
+
+  if (days >= 3 && days < 7) {
+    totalCost -= 20;
+  }
+
+  return totalCost;
```

Comment:
if our condition don't match we just return baseCost
```suggestion
  return baseConst;
```

Reply:
fixed